add kk980 keyboard (#14436)
qmk/qmk_firmware: keyboards/sets3n/kk980/readme.md

# kk980

![kk980](https://imgur.com/a/9JJjC77)

*A short description of the keyboard/project*

* Keyboard Maintainer: [sets3n](https://github.com/sets3n)
* Hardware Supported: *The PCBs, controllers supported*

> * Hardware Availability: *Links to where you can find this hardware*

Reviewer: These should be filled out.

Also, could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Author: Thank you, I learned a lot.

Author: This is very helpful for me.

# kk980

> ![kk980](https://i.imgur.com/MQA6FXb.jpg)

Reviewer 2: This image is sideways.

A hotswap 98% RGB Mechanical keyboard.

* Keyboard Maintainer: [sets3n](https://github.com/sets3n)

> * Hardware Supported: *kk980 PCB*

Reviewer 2: ```suggestion
* Hardware Supported: kk980 PCB
```

* Hardware Availability: -

## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available

## HEX

Make example for this keyboard (after setting up your build environment):

>     `make sets3n/kk980:default`
> 
> ## FLASH
> 
> Flashing example for this keyboard:
> 
>     `make sets3n/kk980:default:flash`

Reviewer 2: ```suggestion
    make sets3n/kk980:default

## FLASH

Flashing example for this keyboard:

    make sets3n/kk980:default:flash
```

# kk980

> ![kk980](https://imgur.com/a/9JJjC77)

Reviewer 2: Please use a direct image link, and fix the orientation.

A hotswap 98% RGB Mechanical keyboard.

* Keyboard Maintainer: [sets3n](https://github.com/sets3n)
* Hardware Supported: *kk980 PCB*
* Hardware Availability: -

## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available

> ## HEX
> 
> Make example for this keyboard (after setting up your build environment):
> 
>     `make kk980:default`
>     `make kk980:via`
> 
> ## FLASH
> 
> Flashing example for this keyboard:
> 
>     `make kk980:default:flash`

Reviewer 2: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make sets3n/kk980:default

Flashing example for this keyboard:

    make sets3n/kk980:default:flash
```
